Platon/backend/app/workspace_system/views.py

```diff
@@ -0,0 +1,577 @@
+    @api.expect(delete_issue_parser)
+    @login_required
+    @workspace_exists(param_loc='form',workspace_id_key='workspace_id')
+    @active_contribution_required(param_loc='form',workspace_id_key='workspace_id')
+    def delete(user_id, self):
+        '''
+            Deletes an issue
+        '''
+        form = DeleteIssuesForm(request.form)
+        if form.validate():
+            
+            try: 
+                # Find issue record
+                issue = Issue.query.filter(Issue.id == form.issue_id.data).first()
+
+                if issue is None:
+                    return make_response(jsonify({'error': 'Issue not found'}), 404)
+
+                db.session.delete(issue)
+                db.session.commit()
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+
+            return make_response(jsonify({'msg': 'Issue is successfully deleted'}), 200)
+
+        else:
+            return make_response(jsonify({'error': 'Input Format Error'}), 400)
+
+@workspace_system_ns.route("/issue/assignee")
+class IssueAssigneeAPI(Resource):
+
+    # pagination will be added later.
+    @api.doc(responses={401 : 'Account Problems', 400 : 'Input Format Error' ,500 : ' Database Connection Error', 404: 'Not found'})
+    @api.response(200, 'Success', issue_assignee_list_model)
+    @api.expect(get_issue_assignee_parser)
+    @login_required
+    @workspace_exists(param_loc='args',workspace_id_key='workspace_id')
+    def get(user_id, self):
+        '''
+            Get Issue Assignees.
+        '''
+        form = GetIssueAssigneeForm(request.args)
+        if form.validate():
+            workspace_id = form.workspace_id.data
+
+            # check if the workspace is public or not.
+            try:
+                workspaceSearch = Workspace.query.filter(Workspace.id == workspace_id).first()
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+
+            if workspaceSearch.is_private:
+                # check if current user is an active contributor.
+                try:
+                    contributionSearch = Contribution.query.filter((Contribution.workspace_id == workspace_id)&(Contribution.user_id == user_id)).first()
+                except:
+                    return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+
+                if contributionSearch is None:
+                    return make_response(jsonify({'error': 'User is not a contributor'}), 401)
+
+                if not contributionSearch.is_active:
+                    return make_response(jsonify({'error': 'User is not an active contributor currently'}), 401)
+            
+            try:
+                issueAssigneeSearch = IssueAssignee.query.filter(IssueAssignee.issue_id == form.issue_id.data)
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
```

Comment:
Same as above.

Reply:
With the latest update, I've added all()